MapGenerator.java

```diff
@@ -0,0 +1,47 @@
+package brickBreaker;
+
+import java.awt.BasicStroke;
+import java.awt.Color;
+import java.awt.Graphics2D;
+
+public class MapGenerator 
+{
+ public int map[][];
+ public int brickWidth;
+ public int brickHeight;
```

Comment:
Shouldn't these members be private?

Reply:
Okay, we made this public so that it can be accessed by the gameplay class

Reply:
It's better to provide public `get` and `set` methods to access and update the value of private variables right?

https://www.w3schools.com/java/java_encapsulation.asp

Reply:
> @SamvruthaShaji Is it possible to use a formatter to automatically align your code, provide proper spaces etc? I've also added a comment regarding accessibility of your class members. LGTM otherwise! Also @Aiswarya-anand, can you comment here what parts you're focusing on?
I am working on another class(gameplay) and jar file

Reply:
You can maybe check out https://www.codejava.net/coding/10-java-core-best-practices-every-java-programmer-should-know for more best practices

Reply:
Yes , we didn't think about that,
Will provide getters and setters

Reply:
Yes , we didn't think about that,
Will provide getters and setters  
Thankyou